isis/tests/Fixtures.h

```diff
@@ -249,6 +249,14 @@ class CSMCameraDemFixture : public CSMCubeFixture {
 
     void SetUp() override;
 };
+
+class HistoryBlob : public TempTestingFiles {
+  protected:
+    Blob *historyBlob;
```

Comment:
This needs to get clean-ed up at the end of each test. I would just change to not be a pointer and assign over it in setup.